ext/opentelemetry-ext-flask/src/opentelemetry/ext/flask/__init__.py

```diff
@@ -158,11 +159,20 @@ class FlaskInstrumentor(BaseInstrumentor):
 
     def __init__(self):
         super().__init__()
-        self._original_flask = None
+        self._original_flask_class = None
 
-    def _instrument(self):
-        self._original_flask = flask.Flask
+    def _instrument(
+        self, flask_class=None
+    ):  # pylint: disable=arguments-differ
+        if flask_class is not None:
+            self._original_flask_class = flask_class
+            return _InstrumentedFlask
+
+        self._original_flask_class = flask.Flask
         flask.Flask = _InstrumentedFlask
 
-    def _uninstrument(self):
-        flask.Flask = self._original_flask
+        return None
```

Comment:
Why is `None` returned if the original class is not supplied? Shouldn't `InstrumentedFlask` be returned?